domains/todolists.is-local.org.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Project about lists your work and mark done when finish and make your work easier than ever",
+
+    "domain": "is-local.org",
+    "subdomain": "todolists",
+
+    "owner": {
+        "email": "thisismegiabao@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "889dba5d-1f7f-4429-9aba-b30ee01cc0b3.id.repl.co",
+        "TXT": ["replit-verify=889dba5d-1f7f-4429-9aba-b30ee01cc0b3"]
```

Comment:
Cannot have a TXT record with CNAME. CNAME records must stand alone without another record.

Reply:
you mean i need to delete the txt?

Reply:
hello again, i have fix again the records, please check again. @DEV-DIBSTER